src/modules/addNewChain/components/ChangeNetwork.tsx

```diff
@@ -0,0 +1,39 @@
+import { FC, useEffect } from 'react';
+
+import { useAccount } from 'hooks';
+
+import { Box, Button, Text } from 'blocks';
+
+export type ChangeNetworkProps = {
+  handleNextStep: () => void;
+  alias: string;
+};
+
+const ChangeNetwork: FC<ChangeNetworkProps> = ({ handleNextStep, alias }) => {
+  const { switchChain, chainId } = useAccount();
+  const aliasChainId = parseInt(alias.split(':')[1]);
```

Comment:
This i used in multiple components. Let's create a function for in common or AddNewChain.utils.ts